typescript/src/processors/mergeParameterInfo.ts

```diff
@@ -0,0 +1,29 @@
+import { DocCollection, Document, Processor } from 'dgeni';
+import { ParameterDoc } from '../api-doc-types/ParameterDoc';
+
+/**
+ * @dgProcessor
+ *
+ * @description
+ * Merge the description from `@param` tags into the parameter docs
+ * extracted from the TypeScript
+ */
+export function mergeParameterInfo() {
+  return new MergeParameterInfoProcessor();
+}
+
+export class MergeParameterInfoProcessor implements Processor {
+  $runAfter = ['readTypeScriptModules', 'tags-extracted'];
+  $runBefore = ['rendering-docs'];
+
+  $process(docs: DocCollection) {
+    docs.forEach((doc: ParameterDoc) => {
+      if (doc.docType === 'parameter') {
+        const paramTag = doc.callableDoc.params && doc.callableDoc.params.find(param => param.name === doc.name);
```

Comment:
Out of curiosity: I'm trying to understand where this `params` property comes from? 

Also I kind of don't like the `callableDoc` name. It would be more clear if it would be something like `callExpressionDoc`.

Reply:
I changed the name to `parameterContainer`

Reply:
Nice, I like that